### ISSUE TITLE
Allow `--genc` and `--list` arguments to be used in the same `sde generate` command

### ISSUE THREAD

**Issue by chrispcampbell** [climateinteractive/SDEverywhere#424]
Currently the `sde generate` command only allows for one "operation" to be performed at a time.  The build package therefore has to run `sde generate` twice (once with `--genc` then again with `--list`) which leads to redundant work.  It would be better if we could make the internal `parseAndGenerate` function take an array of operations instead of a single operation, which would allow for multiple operations to be performed in the same invocation.

**Comment by chrispcampbell** [comment]
Running the transpiler with the new parsing code is already quite a bit faster than with the old.  But by combining `--genc` and `--list` into a single step, our build times are even faster.  Here are some performance numbers measured when building En-ROADS (M1 Max, times in seconds, lower is better):

| parser | --genc only | --list only | before fix | after fix |
| --- | ---: | ---: | ---: | ---: |
| old | 6.8 | 4.4 | 11.2 | 7.6 | 
| new | 4.7 | 4.8 | 9.5 | 5.3 |

In other words, before this fix, we'd perform a lot of redundant work by running `--genc` and `--list` in separate steps (see "before fix", which adds up the first two columns), but now that we can run them in a single command (see "after fix"), we shave off about 4 seconds from the build.
